src/lumigo_tracer/parsers/parser.py

```diff
@@ -94,13 +95,39 @@ def parse_response(self, url: str, status_code: int, headers, body: bytes) -> di
 class DynamoParser(ServerlessAWSParser):
     should_add_message_id = False
 
+    def _extract_message_id(self, body: dict, method: str):
+        if method == "PutItem" and body.get("Item"):
+            return md5hash(body["Item"])
+        elif method in ("UpdateItem", "DeleteItem") and body.get("Key"):
+            return md5hash(body["Key"])
+        elif method == "BatchWriteItem" and body.get("RequestItems"):
+            first_item = next(iter(body["RequestItems"].values()))
+            if first_item:
+                if first_item[0].get("PutRequest"):
+                    return md5hash(first_item[0]["PutRequest"]["Item"])
```

Comment:
Are we sure it will have `"Item"`? If not, it may raise an exception.

Reply:
If, otherwise boto3 will crash